.github/workflows/docker.yml

```
@@ -1,52 +1,45 @@
-            type=raw,value=latest
-            type=ref,event=tag
-      
-      - 
-        name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
+    push_to_registry:
+        name: Push Docker image to Docker Hub
+        runs-on: ubuntu-latest
+        steps:
+            - name: Check out the repo
+              uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)